src/plugins/camera/camera_impl.cpp

```diff
@@ -120,6 +120,14 @@ void CameraImpl::deinit()
     _camera_found = false;
 }
 
+void CameraImpl::prepare()
+{
+    // This is a trick to get mavsdk_server to initialize the camera plugin. Because mavsdk_server
+    // only initializes plugins when they are first used (LazyPlugin), and this may end up timing
+    // out on the first call of a camera function. By calling this before, one can ensure that the
+    // camera plugin is initialized early.
```

Comment:
Oh funny, you don't actually do anything in here. I thought you would kick off something.

Reply:
Yeah I thought it was clever, but you're right. I changed it to actually return `Success` when the camera_definition has been downloaded.

Reply:
And it will retry downloading the camera_definition every time you call `prepare()`.